zooniverse/front-end-monorepo
# lib-classifier: Add disableImageToolbar view to SubjectViewerStore, refactor toolbar buttons

## Package
- lib-classifier

## Linked Issue and/or Talk Post
- towards #3093, iteration on #4539

## Describe your changes
- adds `disableImageToolbar` view to SubjectViewerStore
- refactors image related toolbar buttons to be disabled per `disableImageToolbar`
- refactors toolbar buttons styling if disabled
- adds SubjectViewerStore tests for `disableImageToolbar`
- refactor related stories

## How to Review
_Helpful explanations that will make your reviewer happy:_
- What Zooniverse project should my reviewer use to review UX?
  - local: https://local.zooniverse.org:8080/?project=1693&workflow=3430
  - fe-project-branch: TBD
- What user actions should my reviewer step through to review this PR?
  - load subject viewer with image, note image related buttons work
  - load subject viewer with text or video, note image related buttons are disabled
- Which storybook stories should be reviewed?
  - all ImageToolbar stories
  - [new ImageToolbar story](http://localhost:6006/?path=/story/image-toolbar-image-toolbar--with-single-text-viewer) with text viewer and disabled image related buttons
- Are there plans for follow up PR’s to further fix this bug or develop this feature? maybe, depending on user feedback

# Checklist
_PR Creator - Please cater the checklist to fit the review needed for your code changes._
_PR Reviewer - Use the checklist during your review. Each point should be checkmarked or discussed before PR approval._

## General

## General UX
Example Staging Project: [i-fancy-cats](https://local.zooniverse.org:3000/projects/brooke/i-fancy-cats)
  - Can be used with a screen reader [BBC guide to VoiceOver](https://bbc.github.io/accessibility-news-and-you/assistive-technology/testing-steps/voiceover-mac.html)
  - Can be used from the keyboard [WebAIM guide to keyboard testing](https://webaim.org/techniques/keyboard/#testing)
  - It is passing accessibility checks in the storybook

## Bug Fix

## Refactoring
